tests/grpc/grpc_test.go

```diff
@@ -227,3 +227,49 @@ func Test_Grpc_Terminate_Recursive(t *testing.T) {
 		})
 	}
 }
+
+func Test_Grpc_ReuseInstanceIDSkipOrTerminate(t *testing.T) {
+	delayTime := 4 * time.Second
+	r := task.NewTaskRegistry()
+	r.AddOrchestratorN("SingleActivity", func(ctx *task.OrchestrationContext) (any, error) {
+		var input string
+		if err := ctx.GetInput(&input); err != nil {
+			return nil, err
+		}
+		var output string
+		ctx.CreateTimer(delayTime).Await(nil)
+		err := ctx.CallActivity("SayHello", task.WithActivityInput(input)).Await(&output)
+		return output, err
+	})
+	r.AddActivityN("SayHello", func(ctx task.ActivityContext) (any, error) {
+		var name string
+		if err := ctx.GetInput(&name); err != nil {
+			return nil, err
+		}
+		return fmt.Sprintf("Hello, %s!", name), nil
+	})
+
+	cancelListener := startGrpcListener(t, r)
+	defer cancelListener()
+	instanceIDs := api.InstanceID("SKIP_IF_RUNNING_OR_COMPLETED")
+	ReuseIdOption := &api.OrchestrationIDReuseOption{
+		CreateOrchestrationAction: api.SKIP,
+		OrchestrationStatuses: []api.OrchestrationStatus{
+			api.RUNNING,
+			api.COMPLETED,
+			api.PENDING,
```

Comment:
`SKIP` belongs to enum type `CreateOrchestrationAction`, `RUNNING` belongs to enum type `OrchestrationStatus`, is there a good way to tell they are different enums defined under the same package?